src/lockfreequeues/constants.nim

```diff
@@ -0,0 +1,12 @@
+# lockfreequeues
+# © Copyright 2020 Elijah Shaw-Rutschman
+#
+# See the file "LICENSE", included in this distribution for details about the
+# copyright.
+
+## Constants used by lockfreequeues
+
+const CACHELINE_BYTES* = when defined(powerpc):
+  128 ## The size of a cache line (128 bytes on PowerPC)
```

Comment:
AFAIK some Samsung phones also have a 128 bytes cache line, but getting those detail is a pain.

Also Intel CPU prefetch cache lines by pairs but then 128 bytes is a lot. Personally I don't bother anymore and just use 64 like you.